src/screens/Customers/CustomerProfile.tsx

```diff
@@ -126,35 +157,291 @@ function CustomerProfile(props: CustomerProps) {
           }}
           size="small"
         >
-          <AddIcon style={{ color: 'white' }} />
+          <FlashOnIcon style={{ color: 'white' }} />
         </IconButton>
       </div>
     );
   };
 
+  const getTariffInfo = () => {
+    if (customer.meterType == "Inactive") {
+      tariffInfo = [
+        { number: UNDEFINED_AMOUNT , label: 'Fixed\nTariff', unit: '' },
+        { number: UNDEFINED_AMOUNT, label: 'Unit\nTariff', unit: '' },
+        { number: UNDEFINED_AMOUNT, label: 'Minimum\nUnits', unit: '' },
+      ]
+      return (
+        <div className={classes.tariffInfo}>
+          {tariffInfo.map((info, index) => (
+            <div key={index} className={classes.tariffInfoItem}>
+              <Typography variant="h3" align={'center'} className={classes.gray}>
+                {info.number} {info.unit}
+              </Typography>
+              <Typography variant="body1" align={'center'} className={classes.tariffInfoLabel}>{info.label}</Typography>
+            </div>
+          ))}
+        </div>
+      );
+    } else {
+      return (
+        <div className={classes.tariffInfo}>
+          {tariffInfo.map((info, index) => (
+            <div key={index} className={classes.tariffInfoItem}>
+              <Typography variant="h3" align={'center'}>
+                {info.number} {info.unit}
+              </Typography>
+              <Typography variant="body1" align={'center'} className={classes.tariffInfoLabel}>{info.label}</Typography>
+            </div>
+          ))}
```

Comment:
The `if` and `else` returns are basically the same, so we might want to abstract this away into another component? We could call it `TariffPlanInfo` because Annie made an `InventoryInfo` so let's stick w/that convention?

Reply:
Even better would just to use another ternary on `className`, since that's the only thing you're conditioning on. So something like 
```
<Typography variant="h3" align={'center'} className={customer.meterType == "Inactive" ? classes.gray : ''}> 
```